armi/reactor/converters/blockConverters.py

```diff
@@ -637,6 +639,7 @@ def _buildNthRing(self, pinComponents, ringNum):  # pylint: disable=too-many-loc
             bigComponent = self._addSolidMaterialRing(
                 bcs, innerDiameter, outerDiameter, name
             )
+            bigComponent.p.flags = bcs.p.flags
```

Comment:
Can the transferring of flags be moved into `_addSolidMaterialRing`? That would be easier to track and less error prone if somebody were to use `_addSolidMaterialRing` somewhere else in the future.